js/todoapp.js

```diff
@@ -0,0 +1,171 @@
+const demoTodo = [
+  {
+    name: "Follow Jijuzo on GitHub",
+    id: 1,
+    completed: false,
+  },
+];
+
+//**Todo creation**
+const todos = document.querySelector(".todo-ul");
+function addingTodo(todo) {
+  const newTodo = document.createElement("li");
+  newTodo.className = "mycheck";
+  newTodo.setAttribute("completed", todo.completed);
+  newTodo.innerHTML = `
+            <input id="${todo.id}" type="checkbox" class="check">
+            <label class="task-label" for="${todo.id}">${todo.name}</label>
+            <i class="fa-solid fa-trash-can completed-trash"></i>
+    `;
+  todos.appendChild(newTodo);
+}
+
+// **using demo todo**
+let tasks = JSON.parse(localStorage.getItem("tasks")) || demoTodo;
```

Comment:
`??` operator is a more better here than `||`
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing